cmd/prometheus/main.go

```diff
@@ -871,7 +877,7 @@ func main() {
 			name: "gcm_export",
 			reloader: func(cfg *config.Config) error {
 				// Call in closure to not call Global() before it's initialized below.
-				return gcm_export.Global().ApplyConfig(cfg)
+				return gcm_export.Global().ApplyConfig(cfg, &opts.ExporterOpts)
```

Comment:
question (non-blocking): Ah - IIUC you originally left the arg `nil` as the reload function would never trigger at runtime anyway, since the opts (as of this PR) can only change when the process restarts. Do I have that correct?

Reply:
That's correct!